bigbluebutton-html5/imports/ui/components/video-provider/video-list/styles.scss

```diff
@@ -173,3 +174,23 @@
     font-size: 75%;
   }
 }
+
+.webRTCStats {
+  background: rgba(0,0,0,0.85);
+  padding: 15px;
+  height: 100%;
+  color: #fff;
+  border-radius: 5px;
+  position: absolute;
+  font-size: 12px;
```

Comment:
I think this will overflow on mobile - outside of the webcam

Reply:
It doesn't overflow, it is cropped. But there's too much information for a small space, especially on Chrome. I don't know exactly what's the best layout for mobile.